ldk/javascript/src/types/oliveHelps/index.d.ts

```diff
@@ -290,7 +290,7 @@ declare namespace OliveHelps {
   interface Whisper {
     id: string;
     close: Readable<undefined>;
-    // update(whisper: NewWhisper, cb: (err: string) => void): void // TODO: Implement
+    update(whisper: UpdateWhisper, cb?: (err: Error) => void): void 
```

Comment:
Given the current WIP implementation for "form like behavior" for whispers, the LDK will need a way to intercept components given as `UpdateWhisper` here in order to keep track of their value states.